invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -332,44 +333,19 @@ def get_funding(self, obj):
 
         def _serialize_funder(funding_object):
             """Serializes one funder."""
-            funder = funding_object["funder"]
             award = funding_object.get("award", {})
 
-            funder_string = ""
-
-            if award:
-                identifiers = award.get("identifiers", [])
-                title = award.get("title", {})
-                title = list(title.values())[0] if title else "null"
-                number = award.get("number", "null")
-
-                funder_string += f"award_title={title}; "
-                funder_string += f"award_number={number}; "
-
-                if identifiers:
-                    identifier = identifiers[0]
-                    scheme = identifier.get("scheme", "null")
-                    identifier_value = identifier.get("identifier", "null")
-
-                    funder_string += f"award_identifiers_scheme={scheme}; "
-                    funder_string += (
-                        f"award_identifiers_identifier={identifier_value}; "
-                    )
-
-            funder_id = funder["id"]
-            funder_name = funder.get("name", "null")
-
-            # Serialize funder
-            funder_string += f"funder_id={funder_id}; "
-            funder_string += f"funder_name={funder_name}; "
-
-            return funder_string
+            award_title = award.get("title", {}).get(
+                "en", "null"
```

Comment:
Won't ``null``  lead to potentially getting something like this?

```
    <subfield code="c">null</subfield>
    <subfield code="a">null</subfield>
```

I think it should just leave out the subfield instead